client/src/app.jsx

```diff
@@ -19,6 +18,30 @@ class App extends React.Component {
     }
   }
 
+  componentDidMount() {
+    // ajax request to the server for price data
+    $.ajax({
+      // TODO: fixed stock ticker for now
+      url: `http://localhost:3000/price/${'ABCD'}`,
```

Comment:
Should the ticker be set up as a default instead of hard coded?

Reply:
yes it should, updated.